app/templates/layouts/one-column.html

```diff
@@ -8,6 +8,8 @@
 
 <main id="content" role="main">
 
+  {% block breadcrumbs %}{% endblock %}
```

Comment:
Or it could just go in here, and then be overridden in inheriting templates where necessary